src/app/views/sidebar/Sidebar.tsx

```diff
@@ -5,7 +5,6 @@ import { telemetry } from '../../../telemetry';
 import History from './history/History';
 import SampleQueries from './sample-queries/SampleQueries';
 
-
```

Comment:
Reason for this? 👀 I think double-space is an intended separator here

Reply:
tbh I have no idea（´＠＿＠｀ ）

Reply:
fyi that the Azure Portal codebase I'm working on has tslint checking against double empty lines (and many other rules..)

Reply:
resolved in https://github.com/LokiLabs/microsoft-graph-explorer-v4/pull/16/commits/55666db4d0964581599d5f907b355fb066034d67